docs/configuration/integrations/integrations-next/app-o11y-receiver-config.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "app_o11y_config"
++++
+
+# app_o11y_receiver_config
+
+The `app_o11y_receiver_config` block configures the `app-o11y-receiver`
+integration. This integration exposes a http endpoint that can receive telemetry
+from the [grafana javascript agent](https://github.com/grafana/grafana-javascript-agent)
+and forward it to logs, traces or metrics backends.
+
+Full reference of options:
+
+```yaml
+  autoscrape:
+    # Enables autoscrape of integrations.
+    [enable: <boolean> | default = true]
+
+    # Specifies the metrics instance name to send metrics to. Instance
+    # names are located at metrics.configs[].name from the top-level config.
+    # The instance must exist.
+    #
+    # As it is common to use the name "default" for your primary instance,
+    # we assume the same here.
+    [metrics_instance: <string> | default = "default"]
+
+    # Autoscrape interval and timeout. Defaults are inherited from the global
+    # section of the top-level metrics config.
+    [scrape_interval: <duration> | default = <metrics.global.scrape_interval>]
+    [scrape_timeout: <duration> | default = <metrics.global.scrape_timeout>]
+
+  # Integration instance name
+  [instance: <string>]
+
+  # Domains in which the agent is sending data from. For example "https://myapp.com"
+  cors_allowed_origins:
+    [- <string>]
+
+  # Configure rate limiting. The HTTP server of the App observability implements
+  # a token bucket rate limitng algorithm in which we can configure the maximum RPS
+  # as well as the burstiness (peaks of RPS)
+  rate_limiting:
+    [enabled: <boolean> | default = false]
+    [rps: <number> | default = 100]
+    [burstiness: <number> | default = 50]
+
+  # If configured, incoming requests will be required to specify this key in "x-api-key" header
+  [api_key: <string>]
+
+  # Max allowed payload size in bytes for the JSON payload. Interanlly the
+  # Content-Length header is used to make this check
+  [max_allowed_payload_size: <number> | default = 0]
+
+  # Traces instance to send traces to. This assumes that you have a traces config with such instance defined
+  [traces_instance: <string> | default = ""]
+
+  # Logs instance to send logs and exceptions to. This assumes that you have a logs
+  # config with the instance defined
+  [logs_instance: <string> | default = ""]
+
+  # Server config refres to the HTTP endpoint that the integration will be exposing
+  # to receive data from.
+  server:
```

Comment:
IMO the CORS and rate limiter settings should be moved to this block since they're server-specific.